src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,35 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _is_illustration_metadata_name(self, name: str) -> bool:
```

Comment:
You are using it once and it's a very simple check ; why the extra function?

Reply:
Giving it a name is a way of making the code a little more self-documenting. Really the prefix should be put somewhere shared and reused as well, but I didn't want to edit all the other places the constant is used in this change.